strings/anagram/isanagram.go

```diff
@@ -0,0 +1,41 @@
+// isanagram.go
```

Comment:
Use a go package comment instead of this comment style.

Reply:
Agreed

Reply:
Using a multi-line comment is not what I meant. See how to write go documentation comments: https://go.dev/doc/effective_go#commentary